src/choose.py

```diff
@@ -29,16 +30,19 @@
 '''
 
 
-def doProgram(stdscr, flags, cursesAPI=None, lineObjs=None):
+def doProgram(stdscr, flags, keyBindings=None, cursesAPI=None, lineObjs=None):
```

Comment:
If I'm understanding well, this translates to `def doProgram(stdscr, flags, keyBindings, ...)`; I think it's a good idea.

Yes, the Controller (which is the "center" class) definitely needs it.

There is also the interesting, open, idea that the KeyBindings class (and its backing file) is in reality nothing else than a generic configuration class.
In the future, it will be very easy to convert it to a more general `Configuration` (or any other name).

Now, I'm saying this to strenghten the point that it's worth making required, being a configuration.